src/components/displayDocuments/documentCard.tsx

```diff
@@ -44,14 +45,7 @@ const DocumentDetails = styled.div`
   line-height: 19px;
 `;
 
-interface DocumentCardProps {
-  className?: string;
-  title: string;
-  url: string;
-  image?: string | undefined;
-}
-
-export const DocumentCard: React.FunctionComponent<DocumentCardProps> = ({ image, url, title }) => {
+export const DocumentCard: React.FunctionComponent<Document> = ({ imageName, url, title }) => {
   const [qr, setQr] = useState(false);
```

Comment:
Will be better to name this as `showQrView` and `setShowQrView`. I was thinking if QR code was dynamically generated when i saw this.